### ISSUE TITLE
Weird static flickering esp32-4827S043C

### ISSUE THREAD

**Issue by SquareShox** [rzeldent/esp32-smartdisplay#188]
https://github.com/user-attachments/assets/99a6d043-01e8-418e-91c4-ced995b3e36a

Hello im using esp32-4827S043C and i have this weird static screen flickering . ive tried changing the frequency of screen clock, lvgl handler etc nothing seems to work . Maybe it is a hardware issue but this occurs on my other screen (same model). Can someone help?

Issue does not occur when default sofware is running

**Comment by rzeldent** [comment]
Hi,

Looks indeed like some interference / clock setting...
Does this also happens without WiFi enabled?

**Comment by SquareShox** [comment]
Yes i use standard clock configuration now but this happens even in demo examples of library

**Comment by 3cky** [comment]
Had the same flickering problem with ESP32-4827S043C board with clock frequencies as low as 4 MHz. For me, it was fixed by setting `ST7262_PANEL_CONFIG_DE_GPIO_NUM=-1`.

**Comment by rzeldent** [comment]
Do you think a PR is handy to do so it will be available in the repo itself?
Are you using the main or develop branch?
